src/internal/m365/backup.go

```diff
@@ -156,6 +156,17 @@ func (ctrl *Controller) IsBackupRunnable(
 	service path.ServiceType,
 	resourceOwner string,
 ) (bool, error) {
+	if service == path.GroupsService {
+		_, err := ctrl.AC.Groups().GetByID(ctx, resourceOwner)
+		if err != nil {
+			// TODO(meain): check for error message in case groups are
```

Comment:
This is pretty important: when can we expect this todo to get pulled in?

Reply:
This will be a close follow up to this. I mostly just wanted to get things working first and then fix up all the gotchas.